test/core/gprpp/inlined_vector_test.cc

```diff
@@ -94,7 +94,7 @@ TEST(InlinedVectorTest, ConstIndexOperator) {
     v.push_back(i);
     EXPECT_EQ(i + 1UL, v.size());
   }
-  auto const_func = [kNumElements](const InlinedVector<int, 5>& v) {
```

Comment:
doesn't need to be captured because `kNumElements` it's a const, and this satisfies...
> A lambda expression can read the value of a variable without capturing it if the variable (...) has const non-volatile integral or enumeration type and has been initialized with a constant expression

See http://en.cppreference.com/w/cpp/language/lambda